index.html

```diff
@@ -5,12 +5,94 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Gwendolyn&family=Open+Sans:wght@300;500&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
 </head>
-
 <body>
-    <!-- Add your markup here -->
+<div class="wrapper">
+    <header class="header">
+        <nav class="menu">
+            <div class="menu_icon">
```

Comment:
For accessibility, you should probably make this a `<button>`. That way screen readers identify it as such.